doc/getting_going.rst

```diff
@@ -45,15 +45,24 @@ Dependencies
 PSyclone is written in Python so needs Python to be installed on the
 target machine. PSyclone has been tested under Python 2.6.5 and 2.7.3.
 
-PSyclone immediately relies on two external Python packages; fparser
-and pyparsing. In addition, fparser requires numpy. If the visual
-representation of a schedule's dependency graph is required then the
-python package graphviz is necessary to provide python bindings as well
-as the graphviz package itself.  In order to run the test suite py.test is
-required. The easiest way to satisfy the Python dependencies is to use
-the Python Package Index (pypi.org) and ``pip``. See
+PSyclone immediately relies on two external Python packages;
+``fparser`` and ``pyparsing``. In addition, ``fparser`` requires
+``numpy``. In order to run the test suite ``py.test`` is required. The
+easiest way to satisfy the Python dependencies is to use the Python
+Package Index (pypi.org) and ``pip``. See
 https://packaging.python.org/installing/ for more information.
 
+In addition to the mandatory dependencies just described, PSyclone
+also has optional dependencies on both ``graphviz`` and ``termcolor``.
+PSyclone can use graphviz to produce a visualisation of a schedule's
+dependency graph. If this is desired then the Python package ``graphviz``
+(for the Python bindings) as well as the graphviz package itself must
+be installed. The Python package ``termcolor`` is used for pretty-printing
+a schedule in terminals that support coloured text. If the package is
```

Comment:
Probably worth saying what happens if graphviz is not available seeing you do so for termcolor.

Reply:
Done.